app/managers.py

```diff
@@ -0,0 +1,57 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("actors.db")
+        self.create_table()
+
+    def create_table(self) -> None:
+        self._connection.execute(
+            """
+            CREATE TABLE IF NOT EXISTS actors (
```

Comment:
bad sql formatting. You can use this to make it pretty https://sqlformat.org/
Apply everywhere

Reply:
actually here you can keep as is - sql formatter will spread lines too much. But check other queries - the least you can do is to put each keyword on new line - thats the standard